modules/ROOT/pages/comments-commands-events-apis.adoc

```diff
@@ -10,3 +10,30 @@
 The Comments plugin provides the following {productname} commands.
 
 include::partial$commands/comments-cmds.adoc[]
+
+== API
+
+The Comments plugin exposes an API, `editor.plugins.tinycomments.getEventLog()`, which returns a log of all `CommentChange` events which are fired when:
```

Comment:
This also doesn't follow the standard template for this... take a look at https://www.tiny.cloud/docs/tinymce/6/a11ychecker/#apis. We should have a heading for each API so it shows in the nav at the very least.